src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerFlyoutPopupPanel.iOS.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
```

Comment:
there seems to be missing #if __IOS__

Reply:
It doesn't seem to be required: https://github.com/unoplatform/uno/blob/7a51c1529af8b3f63c3049dbb474fb23fdc19e32/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs#L1
https://github.com/unoplatform/uno/blob/26c5cc5992cae3c8c25adf51eb77ca4b0dd34e93/src/Uno.UI/UI/Xaml/Controls/Button/RadioButton.Android.cs#L1

Reply:
No, it's not required in the Uno.UI project, as long as the file has the correct suffix.